backend/src/services/integration-auth/integration-sync-secret.ts

```diff
@@ -4021,6 +4095,14 @@ export const syncIntegrationSecrets = async ({
         accessToken
       });
       break;
+    case Integrations.DATABRICKS:
+      await syncSecretsDatabricks({
+        integration,
+        integrationAuth,
+        secrets,
+        accessToken
```

Comment:
pass databricks workspace url

Reply:
actually it's passed through integrationAuth here I think